src/software/simulated_tests/simulated_play_test_fixture_test.cpp

```diff
@@ -185,12 +194,18 @@ TEST_F(
     // test. To do this we check the timestamp very close to the test timeout
     auto failing_validation_function = [](std::shared_ptr<World> world_ptr,
                                           ValidationCoroutine::push_type& yield) {
-        EXPECT_LT(world_ptr->getMostRecentTimestamp(), Timestamp::fromSeconds(0.5));
+        if (world_ptr->getMostRecentTimestamp() >= Timestamp::fromSeconds(0.5))
```

Comment:
I think we should get rid of this test altogether. The name of this test is "test_failing_gtest_expect_statement_in_non_terminating_validation_function_causes_test_to_fail" and this test isn't using gtest

We should also remove all the other tests that involve gtest if we're using another way to display error messages

Reply:
yeah good point